wiz-admission-controller/values.yaml

```
@@ -49,6 +49,8 @@ image:
   tag: ""
 imagePullSecrets: []
 
+priorityClassName: ""
```

```suggestion
priorityClassName: "system-cluster-critical"
```

I think your suggestion to set "system-cluster-critical" as the default value is good

---

I have accepted the suggestion.  We should note for semver versioning/release purposes that this will change the default behavior for anyone who upgrades.